tests/v1/test_distributions.py

```diff
@@ -0,0 +1,51 @@
+from itertools import product
+
+import numpy as np
+import pytest
+from scipy.integrate import cumulative_trapezoid
+from scipy.stats import kstest
+
+from petab.v1.distributions import *
+from petab.v2.C import *
+
+
+@pytest.mark.parametrize(
+    "distribution, transform",
+    list(
+        product(
+            [
+                Normal(2, 1),
+                Normal(2, 1, log=True),
+                Normal(2, 1, log=10),
+                Uniform(2, 4),
+                Uniform(-2, 4, log=True),
+                Uniform(2, 4, log=10),
+                Laplace(1, 2),
+                Laplace(1, 0.5, log=True),
+            ],
+            [LIN, LOG, LOG10],
+        )
+    ),
+)
+def test_sample_matches_pdf(distribution, transform):
+    """Test that the sample matches the PDF."""
+    np.random.seed(1)
+    N_SAMPLES = 10_000
+    distribution.transform = transform
+    sample = distribution.sample(N_SAMPLES)
+
+    # pdf -> cdf
+    def cdf(x):
+        return cumulative_trapezoid(distribution.pdf(x), x)
```

Comment:
Makes sense to ensure that the samples match the PDF within PEtab. For the `LIN` `transform` cases, could you additionally compare to `scipy.stats.(log)(uniform/normal/laplace)`? I guess the `LOG` and `LOG10` `transform`s will be dropped in v2 so less important to test I guess